be1-go/channel/consensus/mod.go

```diff
@@ -40,12 +40,12 @@ type Channel struct {
 }
 
 // NewChannel returns a new initialized consensus channel
-func NewChannel(channelID string, hub channel.HubFunctionalities, log zerolog.Logger) Channel {
+func NewChannel(channelID string, hub channel.HubFunctionalities, log zerolog.Logger) channel.Channel {
```

Comment:
I'm curious about the advantages of returning an interface instead of a consensus.Channel. Should we return an interface for all channels?

Reply:
It was just easier to copy what was done for the lao channel than to adapt the tests I used as a benchmark to help me